ToolbarActionButton.tsx

```diff
@@ -0,0 +1,29 @@
+import { Platform, Pressable, StyleSheet, Text } from "react-native";
+import React from "react";
+
+export const ToolbarActionButton = ({
```

Comment:
Do you think we should add some documentation for these types? I'm thinking that they would be useful if new people join the lib development.

Reply:
Do you mean like code comments? If code comments, frankly, I wouldn't add them. I think the code, especially UI widgets, should be easy to understand without comments - IMO the effort of maintaining comments is higher than using them.

Reply:
😅 I generally think it is easier to read the header(comment) of a file to know if it fit my needs rather than lookit at the code. 

But 👍